src/lib/DALMeteorMethods/UserMethods.js

```diff
@@ -1,19 +1,104 @@
 /*eslint-disable*/
 Users = new Mongo.Collection.get("users");
 if (Meteor.isServer) {
-    Meteor.publish("users", function(){
-      return Users.find({_id:this.userId});
+    Meteor.publish("users", function() {
+        return Users.find({
+            _id: this.userId
+        });
     });
-}else {
-  Meteor.subscribe("users");
+    Meteor.publish("usersInRoom", function() {
+        return Users.find({});
```

Comment:
A possible solution to this would be to either make a new collection, only only publish the data you need (user.rooms for users who are currently in my rooms, which is a valid publish, we just need to make sure not to publish the other stuff)